src/EditorFeatures/Core.Wpf/InlineDiagnostics/InlineDiagnosticsAdornmentManager.cs

```diff
@@ -116,47 +116,6 @@ private TextFormattingRunProperties GetFormat(IClassificationType classification
             return _formatMap.GetTextProperties(classificationType);
         }
 
-        /// <summary>
-        /// Get the spans located on each line so that it can only display the first one that appears on the line
-        /// </summary>
-        private void AddSpansOnEachLine(NormalizedSnapshotSpanCollection changedSpanCollection,
```

Comment:
Is this change just moving the method? Can you undo if that's the only change to this file?

Reply:
yeah I just moved the method since it should've been below the calling method. Do I need to create a separate PR for that?